docs/source/getting-started/content/proc_setting-up-codeready-containers.adoc

```diff
@@ -0,0 +1,20 @@
+[id="setting-up-codeready-containers_{context}"]
+= Setting up {prod}
+
+The [command]`{bin} setup` command performs operations to set up the environment of your host machine for the {prod} virtual machine.
+
+This procedure will create the [filename]`~/.crc` directory if it does not already exist.
+
+.Prerequisites
+
+// TODO: Verify that we need `sudo` permissions.
+* Your user account must have permission to use the [command]`sudo` command.
```

Comment:
What other prerequisites are there? Virtualization (`libvirt`) packages? Ensure we have services running (`systemctl is-active libvirtd`)?

Reply:
@gbraad @praveenkumar WDYT?

Reply:
the service is checked by the pre-flight checks: https://github.com/code-ready/crc/blob/master/pkg/crc/preflight/preflight_checks_linux.go#L178-L206 so we can mention the need to have these packages and that the setup sets the enabled, but this is not something that the user needs to handle.

Reply:
Ah, looks like the preflight takes care of all of these questions, actually -- in that case, I think we can just list `sudo` permissions as the only prerequisite. :)

Reply:
> We could mention the packages for other distributions.

right, but this will be resolved in future versions... as we would like to detect and attempt to fix for other distros... and when not possible, suggest a possible 'manual solution' or the docs.